setup.py

```diff
@@ -61,6 +61,7 @@
         'requests',
         'setuptools',
         'tarjan',
+        'plone.app.jquery >= 1.9.1',
```

Comment:
This breaks Plone 4.3 support.

Why did you choose `1.9.1`?
You are introducing the use of [`.on`](http://api.jquery.com/on/), which was introduced in jQuery 1.7, which was packaged in `plone.app.jquery = 1.7.2.1`, the version used for Plone `4.3`. Therefore I assume that this JavaScript will run with `plone.app.jquery = 1.7.2.1`.

Actually, I would prefer to not have a version constraint here at all.
The reason is that we sometimes use `ftw.upgrade` for upgrading very old installations.
Although we no longer guarantee compatibility with older Plone versions it might work anyway as long as we are not blocking compatibility by using version constraints.

I therefore propose to just remove this line. Do you agree?